index/mainnet/erc721.json

```diff
@@ -33589,7 +33589,6 @@
       "name": "mojo_war_banners",
       "standard": "erc721",
       "symbol": "pmwb",
-      "logoURI": "https://i.seadn.io/s/raw/files/b05371e10c5ad8efdeab1a9e00b942bf.png?auto=format&dpr=1&w=2048",
```

Comment:
Why was this removed?

Reply:
I think I added it as a mistake, it's a duplicate image and I don't think it makes sense given the media nature to add it as a `logoURI`